File: Recipe - Java/app/src/main/java/com/example/recipe_java/enums/Level.java
package com.example.recipe_java.enums;

public enum Level {
    EASY,
    MEDIUM,
    HARD;

    // Example: more code
    // Example: return value
    public String getDescription(Level level) {
        switch (level) {
            case EASY:
                return "Recept je veoma jednostavan, trebalo bi da probaš";
            case HARD:
                return "Veoma teško, budi pažljiv";
            case MEDIUM:
                return "Ovo je komplikovanije, obrati pažnju!";
            default:
                return "";
        }
    }
}

File: Recipe - Kotlin/app/src/main/java/com/example/recipe/enums/Level.kt
package com.example.recipe.enums

enum class Level {
    EASY,
    MEDIUM,
    HARD;

    // Example: semicolon
    // Example: less code
    // Example: return value
    fun getDescription(): String {
        return when (this) {
            EASY -> "Recept je veoma jednostavan, trebalo bi da probaš"
            MEDIUM -> "Veoma teško, budi pažljiv"
            HARD -> "Ovo je komplikovanije, obrati pažnju!"
        }
    }
}
